tests/common.go

```diff
@@ -4035,7 +4136,45 @@ func DeleteAzureBucket(bucketName string) {
 		fmt.Sprintf("Failed to delete container. Error: [%v]", err))
 }
 
-// DeleteBucket deletes bucket from the cloud
+// DeleteNfsSubPath delete subpath from nfs shared path.
+func DeleteNfsSubPath() {
+	// Get NFS share details from ENV variables.
+	nfsServerAddress, nfsPath, nfsSubPath, _ := GetNfsCredsFromEnv()
+
+	// Mount the NFS share to the master node.
+	masterNode := node.GetMasterNodes()[0]
+	mountCmds := []string{
+		"mkdir -p /tmp/nfsMount",
+		fmt.Sprintf("mount -t nfs %s:%s /tmp/nfsMount", nfsServerAddress, nfsPath),
+	}
+	for _, cmd := range mountCmds {
+		err := runCmd(cmd, masterNode)
+		expect(err).NotTo(haveOccurred(),
+			fmt.Sprintf("Failed to run [%s] command on node [%s], error : [%s]", cmd, masterNode, err))
+	}
+
+	defer func() {
+		// Unmount the NFS share from the master node.
+		umountCmds := []string{
+			"umount /tmp/nfsMount",
```

Comment:
Why the paths hard coded here ?

Reply:
This is temporaray directory created to mount the NFS path used in backup to master node, and to access the path and delete subpath within, the script will remove the path at the end of function call itself.

Reply:
@trenukarya-px , Thanks for suggestion, I've made the mount directory a constant paramater.